### ISSUE TITLE
Allow a slide to be hidden

### ISSUE THREAD

**Issue by MartinPacker** [MartinPacker/md2pptx#160]
Per [this issue with python-pptx](https://github.com/scanny/python-pptx/issues/319#issuecomment-2193786390), allow slides to be programmatically hidden. "Hidden" here has the Powerpoint semantic for same.

Possible syntax:

```
<!-- md2pptx: hidden: yes  -->
```

This "dynamic metadata" would start hiding slides.

Similarly

```
<!-- md2pptx: hidden: no  -->
```

would end the hiding.

**Comment by MartinPacker** [comment]
Coded.

**Comment by MartinPacker** [comment]
Documented and shipped in [v4.6](https://github.com/MartinPacker/md2pptx/releases/tag/v4.6).
